tests/e2e/tutorials/tutorialBase.js

```diff
@@ -23,7 +23,13 @@ class TutorialBase {
   }
 
   initScreenshoter() {
-    utils.createScreenshotsDir();
+    try {
+      utils.createScreenshotsDir();
```

Comment:
why would it fail creating an empty directory  in CI?

Reply:
Permissions, wrong path... It's just a matter of keeping everything under try/catches. It doesn't hurt, does it?